Bronco_Tree/tree.pde

```diff
@@ -51,16 +54,22 @@ class Tree {
     // Function: grow
     // Description: This function is the heart of the algorithm. grow() looks at each
     // 		    leaf and
-    for (Leaf l : leaves) {
+    for (int i = leaves.size() - 1; i >= 0; i--) {
+      Leaf l = leaves.get(i);
       Branch closest = null; // closest branch to Leaf l
       PVector closestDir = null;
       float record = -1;
 
       for (Branch b : branches) {
```

Comment:
I think we should rename some variables to make it clearer. I'm a fan of longer variable names with more description